src/ex6/client/src/components/todo-list/todoList.jsx

```diff
@@ -0,0 +1,45 @@
+
+import Item from "../todo-item/todoItem.jsx";
+import React from "react";
+import PropTypes from "prop-types";
+import ItemClient from "../../services/item_client.js";
+import { List} from "monday-ui-react-core";
+
+const TodoList = ({setTodoList,items}) => {
+  const itemClient = new ItemClient();
+
+  const onDoneClick = async (id,status) => {
+    const items=await itemClient.changeItemTodo(id,status);
+    setTodoList(items);
+  };
+
+  const deleteTodo = async (id) => {
+    await itemClient.deleteTodo(id);
+    const itemsFilter=items.filter((item) => item.item_id!==id);
+    setTodoList(itemsFilter);
+  };
+
+  return (
+    <List>
+      {items.map((item) => (

```

Comment:
when using map to render a list of objects, react expects a 'key' property to better optimize the page
you also get this error on the page 
<img width="333" alt="image" src="https://user-images.githubusercontent.com/36815567/177956955-972c0809-ad2f-4083-9beb-e323714b8fb6.png">
adding the property
key = {item.item_id}
would have solved it